internal/policy/policy.go

```diff
@@ -675,23 +674,25 @@ func (s *State) findDelegationEntry(roleName string) (tuf.Delegation, error) {
 		delegationTargetsMetadata[name] = targetsMetadata
 	}
 
-	delegationsQueue := topLevelTargetsMetadata.Delegations.Roles
+	delegationsStack := topLevelTargetsMetadata.Delegations.Roles
```

Comment:
It's not really a stack. It's not a typical queue either but it's closer IMO. Note that we may add to the front but we still add multiple entries that are processed in order.

Reply:
Life may be simpler if we avoid `queue`/`stack` in the name hehe. (I'm aware I started using that in the name in the first place)

Reply:
I changed it to delegationsToCheck